pkg/logging/kafka/kafka_integration_test.go

```diff
@@ -355,6 +378,11 @@ Version: 1
 		Format version: 2
 		Response condition: Prevent default logging
 		Placement: none
+		Parse log key-values: false
+		Max batch size: 0
+		SASL authentication method: `+`
+		SASL authentication username: `+`
+		SASL authentication password: `+`
```

Comment:
question - I'm confused by the `\`+\`` can you explain?

Is it to append a trailing whitespace or something?

I'm assuming that your editor is trimming it or something?

Reply:
Yes, this was evidently my editor doing this. I've removed this now.